pkg/console/install_panels.go

```diff
@@ -1050,7 +1050,12 @@ func addInstallPanel(c *Console) error {
 				printToInstallPanel(c.Gui, err.Error())
 				return
 			}
-			doInstall(c.Gui, toCloudConfig(c.config))
+
+			webhooks, err := PrepareWebhooks(c.config.Webhooks, getWebhookContext(c.config))
+			if err != nil {
+				printToInstallPanel(c.Gui, fmt.Sprintf("invalid webhooks: %s", err))
+			}
```

Comment:
Seems like the PrepareWebhooks function never returns a non-nil error

Reply:
Fixed. Log the error message and also print it to the install panel.